Source/OxyPlot/OxyPlot_SL5.csproj

```diff
@@ -30,7 +30,7 @@
     <Optimize>false</Optimize>
     <OutputPath>Bin\Debug\SL5\</OutputPath>
     <IntermediateOutputPath>obj\Debug\SL5\</IntermediateOutputPath>
-    <DefineConstants>DEBUG;TRACE;SILVERLIGHT</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;SILVERLIGHT;SL5</DefineConstants>
```

Comment:
But even then, WP80 / WP80SL also define the SILVERLIGHT definition so I still think SL5 is a good define constant.